src/lib/toolchain_test.rs

```diff
@@ -1,46 +1,59 @@
 use super::*;
+use crate::test;
+
+#[test]
+#[should_panic]
+fn invalid_toolchain() {
```

Comment:
lets rename this to
wrap_command_invalid_toolchain
this makes it much easier for me to understand which test failed based on name.